src/js/components/programs.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable import/prefer-default-export */
+import { newText, newMultimedia, newContainer } from './helpers';
+
+/**
+ * @param {array[nodesHTMl]} cardsContainer es el array de nodosHTML que contiene el
+ *  ul los li y las cards
+ * crea unos divs invisibles que serviran para mover las cards del slider dando clic
+ * agrega la funcionalidad para que al darle clic a los divs invisibles de scroll tanto
+ *  a derecha como a izquierda
+ */
+const createControls = (cardsContainer) => {
+  const controlLeft = newContainer('div', [], ['slider__control', 'slider__control--left']);
+  const controlRight = newContainer('div', [], ['slider__control', 'slider__control--right']);
+
+  controlLeft.addEventListener('click', () => {
```

Comment:
I think the click functionality should be applied to each image, not to an invisible div

Reply:
I refactored the click functionality for slider images